modules/nf-core/bedtools/unionbedg/main.nf

```diff
@@ -0,0 +1,42 @@
+process BEDTOOLS_UNIONBEDG {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::bedtools=2.30.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bedtools:2.30.0--h7d7f7ad_2':
+        'quay.io/biocontainers/bedtools:2.30.0--h7d7f7ad_2' }"
+
+    input:
+    tuple val(meta), path(bedgraph, stageAs: "?/*")
```

Comment:
Why do you use stageAs here?

Reply:
because without it-tests fail

Reply:
The unionbedg gets multiple files, likr samtools merge, I copied from there

Reply:
Aah I get it, we had a lot of discussions in the nf-core community about `stageAs` and the consensus is to avoid it as much as possible and make sure all your input files are names differently :). You can always rename the input files using the `.collectFile()` operator

Reply:
There is only one bedgraph file in test-datasets so @FriederikeHanssen  suggested I will use the same file twice... Seems problematic.

So can I add another bedGraph file to test-datasets? Just the same bedGraph with different values or even the same bedGraph with a different name?

Reply:
I would just use the same Bedgraph file and rename it :) That's the least effort solution

Reply:
I created a PR: https://github.com/nf-core/test-datasets/pull/811
And updated the module to use the new bedGraph, but all tests fails because this file doesn't exists yet in the main test-datasets repository, so I'll be happy if you can review this PR

Reply:
OK, now all tests passed.
Thank you!